src/views/Detail.vue

```diff
@@ -128,6 +152,22 @@ export default {
         .catch(console.error);
     }
   },
+  mounted() {
+    window.addEventListener('keydown', (event) => {
+      if (event.defaultPrevented) {
+        return; // Do nothing if event already handled
+      }
+      switch (event.code) {
+        case 'ArrowLeft':
+          this.carouselIndex -= 1;
+          break;
+        case 'ArrowRight':
+          this.carouselIndex += 1;
+          break;
+        default:
+      }
+    });
+  },
```

Comment:
f you register events here, you will need to unregister them, otherwise they'll linger, causing the DOM events to linger, with events connected they won't be garbage collected.

Here are some resources on the topic:
- https://www.raymondcamden.com/2019/08/12/working-with-the-keyboard-in-your-vue-app
- https://github.com/shentao/vue-global-events
- https://github.com/lupas/vue-keypress

Reply:
In the future, it probably makes sense to put fixups like the PDF one in a separate commit and PR, so we can fast-forward that. Now the PDF-fix is blocked by the arrow keys (no biggie in this case, but for more serious production situations can be a complicating factor).

Reply:
yeah i forgot the event handle remover. I realized in the last meeting. Added them now.